prow/cluster/jobs/istio-private/istio/istio-private.istio.master.gen.yaml

```diff
@@ -71,6 +71,53 @@ postsubmits:
           privileged: true
       nodeSelector:
         testing: test-pool
+  - annotations:
```

Comment:
Since this will also be used in the _private_ prow cluster, you probably want to have a symbolic link from `prow/cluster/build/buildcache.yaml` -> `prow/cluster/private/buildcache.yaml` to ensure this is deployed in the _private_ cluster as well.

Reply:
Can we just drop this in the private one? it will make things much easier, since it will allow other prow instances to utilize it without a cache as well. For the short term we can blacklist the job, but long term this will be used for most jobs

Reply:
Ill blacklist for now